src/styles/blocks/month.scss

```diff
@@ -0,0 +1,52 @@
+.month {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $box-padding;
+  padding-top: 50px;
+  width: (7 * $day-width) + ($box-padding * 2);
+
+  &__day {
+    width: $day-width;
+    height: $day-height;
+    border: 1px solid;
+    cursor: pointer;
+    background-color: #eee;
+    transform: translateY(0);
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @include hover;
+
+    @for $i from 1 through 31 {
+      &:nth-child(n + #{$i}) {
+        &::before {
+          content: "#{$i}";
+        }
+      }
+    }
+
+    $days: "mon" 0,
+      "tue" 100,
+      "wed" 200,
+      "thu" 300,
+      "fri" 400,
+      "sat" 500,
+      "sun" 600, ;
```

Comment:
Move it to the variables file

Reply:
https://github.com/mate-academy/layout_calendar/pull/2728/files#r1328691987
didn't fix it